src/MSBuild/Resources/Strings.resx

```diff
@@ -1661,6 +1661,9 @@
     <value>succeeded: {0}</value>
     <comment>{0} whole number</comment>
   </data>
+  <data name="LongPathsEnabled" xml:space="preserve">
+    <value>LongPathsEnabled = {0}</value>
```

Comment:
A minor rephrase, and it's usually a good idea to explain the parameters to the localizers (otherwise you can get very weird translations when they have to guess what kind of thing might go in that spot):

```suggestion
    <value>Windows LongPathsEnabled registry key = {0}</value>
    
    <comment>"Windows" is the OS, "LongPathsEnabled" should not be localized, and {0} will be boolean true/false</comment>
```